Sources/Rosetta/CardSets/Expert1CardsGen.cpp

```diff
@@ -170,6 +170,20 @@ void Expert1CardsGen::AddPriest(std::map<std::string, Power>& cards)
     power.ClearData();
     power.AddPowerTask(new HealTask(EntityType::ALL_MINIONS, 4));
     cards.emplace("EX1_621", power);
+
+    // ----------------------------------------- SPELL - PRIEST
+    // [EX1_624] Holy Fire - COST:6
+    // - Faction: Priest, Set: Expert1, Rarity: Rare
+    // --------------------------------------------------------
+    // Text: Deal $5 damage. Restore #5 Health to your hero.
+    // --------------------------------------------------------
+    // PlayReq:
+    // - REQ_TARGET_TO_PLAY = 0
+    // --------------------------------------------------------
+    power.ClearData();
+    power.AddPowerTask(new DamageTask(EntityType::TARGET, 5, false));
```

Comment:
`Holy Fire` is spell damage. You should set third parameter to `true`.

Reply:
Thank you! i'll fix it

Reply:
I confirmed it. Resolved.